pkg/utils/common.go

```diff
@@ -391,3 +391,11 @@ func GenerateGroupString(groups []string) string {
 	}
 	return groupString
 }
+
+// GetTolerations returns tolerations from supplied placement policy.
+func GetTolerations(policy *placementv1beta1.PlacementPolicy) []placementv1beta1.Toleration {
```

Comment:
prefer to move this util into apis/placement package, something like,

func (m *ClusterResourcePlacement) Tolerations() []Toleration  {
	....
}

similar to policySnapshot.

We need to get rid of the "utils" package cause the package name is not good and does not provide any meaning.